db/migrate/20170920143824_change_circ_policy_group.rb

```diff
@@ -1,10 +0,0 @@
-class ChangeCircPolicyGroup < ActiveRecord::Migration[5.2]
```

Comment:
Is deleting old migration files safe to do for gems? I know if you delete old migrations in a rails app, you'll have abandoned migration ids when you run something like `bundle exec rake db:migrate:status` (see: https://guides.rubyonrails.org/active_record_migrations.html#old-migrations), and it's generally not recommended. But I don't know how this is handled in gems.

Do you think we should also create a new migration that deletes the table in existing databases?

Reply:
Well ... maybe? I had in the back of my mind that it's a good thing to periodically clear out old migration files -- along the lines of [this article](https://codewithrails.com/clean-up-db-migrations), although that's one I just found now. Except that that says that the one case where you shouldn't prune them is when it's an engine! 

I don't think I've ever run `rake db:migrate:status`, though. 

Probably you're right about creating a new migration.

Reply:
Oh maybe it's a controversial thing - I'm seeing differing opinions online about how to handle old migrations. Some suggestions for deleting old migration files, combining old migration files into one big one (I think maybe this is something we did at a previous place I worked at), moving old migration files into an archived migrations directory, or just leaving them alone to retain history. Since these are old files and our own gem, maybe no real harm in any direction we go in! But a new migration file to remove the table if it exists makes sense to me.

Reply:
Yes ... since this is only used by our one Blacklight app, I think it would be less dangerous than it would be for something shared publicly. But let's go with the more cautious approach. For the sake of not getting more confused, I'm going to close this PR and open a new one.